Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python
+
+WORKDIR /opt/service
+
+RUN pip install poetry
+RUN poetry install
+RUN uvicorn comprehensive_microservice.api_service:app
+
+COPY . .
+
+ENV NAME venv

```

Comment:
why we need that?

Reply:
to create venv in docker image. but probably it doesn't matter

Reply:
you just specify environment variable named NAME and assing 'venv' to it. How does it help to create virtual env?